src/ophys_etl/modules/dff/__main__.py

```diff
@@ -46,22 +56,31 @@ def compute_dff_trace(corrected_fluorescence_trace: np.ndarray,
         `corrected_fluorescence_trace`.
     """
     sigma_f = noise_std(corrected_fluorescence_trace, short_filter_length)
-
+    inactive_trace = corrected_fluorescence_trace.copy()
     # Long timescale median filter for baseline subtraction
-    baseline = medfilt(corrected_fluorescence_trace, long_filter_length)
+    if inactive_percentile > 0 or inactive_percentile < 100:
```

Comment:
This will evaluate true for any value of inactive_percentile due to the or, won't it?

Reply:
Whoops, [my system 1 thinking failed me here 🙃](https://thedecisionlab.com/reference-guide/philosophy/system-1-and-system-2-thinking). I added an additional else case to not use the inactive_percentile filter if it's outside of the bounds.